python/indy-agent.py

```diff
@@ -51,27 +49,23 @@
 
 aiohttp_jinja2.setup(WEBAPP, loader=jinja2.FileSystemLoader('view'))
 
-WEBAPP['msg_router'] = Router()
 WEBAPP['msg_receiver'] = Receiver()
 
 WEBAPP['ui_event_queue'] = UIEventQueue(LOOP)
-WEBAPP['ui_router'] = Router()
 
-WEBAPP['conn_router'] = Router()
 WEBAPP['conn_receiver'] = Receiver()
 
-WEBAPP['agent'] = Agent()
-WEBAPP['modules'] = {
-    'connection': Connection(WEBAPP['agent']),
-    'admin': Admin(WEBAPP['agent']),
-    'admin_walletconnection': AdminWalletConnection(WEBAPP['agent']),
-    'basicmessage': BasicMessage(WEBAPP['agent']),
-    'admin_basicmessage': AdminBasicMessage(WEBAPP['agent'])
-}
-WEBAPP['agent'].modules = WEBAPP['modules']
+AGENT = Agent()
+
+WEBAPP['agent'] = AGENT
+
+AGENT.register_module(Admin(AGENT))
```

Comment:
How do you feel about having the register_module() method instantiate the module internally, passing itself (the agent) as the arg? Would look like this on the outside:
`AGENT.register_module(Admin)`